app/pages/workspaces/workspaces.js

```diff
@@ -291,6 +291,8 @@ export const Workspaces = (props) => {
       : ['Clinic - Workspaces - Go to private workspace'];
 
     trackMetric(...metric);
+    dispatch(actions.sync.setPatientListSearchTextInput(''));
+    dispatch(actions.sync.setIsPatientListVisible(false));
```

Comment:
@krystophv makes sense, and I agree that resetting these filters shouldn't be a concern of `selectClinic`.

Added one more set of reset calls for the Go To Workspace button: a7b65eb3